### ISSUE TITLE
how to add a font

### ISSUE THREAD

**Issue by MACscr** [SVG-Edit/svgedit#327]
how do we add additional fonts? For example, I would love to have the google fonts collection available for use.

**Comment by NeiroNx** [comment]
you can add it here:
https://github.com/SVG-Edit/svgedit/blob/master/editor/svg-editor.html#L359

**Comment by brettz9** [comment]
Having a feature to retain fonts in local storage sounds like a useful feature... Beyond NeiroNx's suggestion, would that meet your needs?

Btw, does anyone know anything about browser support with `@font-face` and SVG `text`?

**Comment by cuixiping** [comment]
Font files can be saved into `IndexedDB`. Adding font files also needs to consider pdf export.

**Comment by brettz9** [comment]
Yeah, IndexedDB does sound good--I was speaking more generically about "local storage" rather than `localStorage`. :) But all something someone else will probably have to implement :)

**Comment by jfhenon** [comment]
Adding fonts may be a useful feature. We would welcome a PR about this.

**Comment by ndunnaoc** [comment]
Any news on this feature?

**Comment by jfhenon** [comment]
Hello, I think we should look into this for version 7.1 (I can't give you a date).
